packages/frontend/src/components/common/modal/ModalActionButtons.tsx

```diff
@@ -22,9 +22,11 @@ const ModalActionButtons = ({
 }: ModalActionButtonsProps) => {
     const [isOpen, setIsOpen] = useState(false);
     const [itemId, setItemId] = useState<string>('');
-    const { mutate: duplicateChart } = useDuplicateMutation(itemId);
-    const { mutate: duplicateDashboard } =
-        useDuplicateDashboardMutation(itemId);
+    const { mutate: duplicateChart } = useDuplicateMutation(itemId, true);
+    const { mutate: duplicateDashboard } = useDuplicateDashboardMutation(
+        itemId,
+        true,
```

Comment:
it looks like dashboards will show the redirectButton but charts will redirect automatically.. is this how it is supposed to work ? both in Explore  and /home pages ?